src/ImageSharp/Processing/Processors/Quantization/QuantizerConstants.cs

```diff
@@ -17,5 +19,20 @@ internal static class QuantizerConstants
         /// The maximum number of colors to use when quantizing an image.
         /// </summary>
         public const int MaxColors = 256;
+
+        /// <summary>
+        /// The minumim dithering scale used to adjust the amount of dither.
+        /// </summary>
+        public const float MinDitherScale = 0;
+
+        /// <summary>
+        /// The max dithering scale used to adjust the amount of dither.
+        /// </summary>
+        public const float MaxDitherScale = 1F;
+
+        /// <summary>
+        /// Gets the default dithering algorithm to use.
+        /// </summary>
+        public static IDither DefaultDither { get; } = KnownDitherings.FloydSteinberg;
```

Comment:
Isn't really a constant. Woldn't it better to define it in `KnownDitherings` instead? (`KnownDitherings.Default`)

Reply:
We'd have to do that across the board for all our `Known***` types.